src/com/aquaq/training/StringEvaluation.java

```diff
@@ -0,0 +1,87 @@
+package com.aquaq.training;
+
+import java.sql.SQLOutput;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class StringEvaluation {
+
+    public char findMaxChar(Scanner scanner){
```

Comment:
Methods should only do one thing and we should aim for method with 10 or less lines.
Reduce this method down by splitting it into other methods.